Datastore/tests/Unit/OperationTest.php

```diff
@@ -536,7 +536,8 @@ public function testRunQueryNoResults()
 
     public function testRunQueryWithReadOptionsFromTransaction()
     {
-        $this->connection->runQuery(Argument::withKey('readOptions'))->willReturn([]);
+        $this->connection->runQuery(Argument::withKey('readOptions'))->willReturn([])
+            ->shouldBeCalled();
```

Comment:
nit: Do you want to add how many no of times this was called? May skip if too much effort for u.

Reply:
I thought to use `shouldBeCalled` instead of `shouldBeCalledOnce` to make the absolute minimum change to the tests to get them to work. This isn't a particularly great reason, so if you think we should use `shouldBeCalledOnce` instead, I'm happy to update them!

Reply:
I prefer `shouldBeCalledOnce`. For individual cases when tests fail, we can leave it as `shouldBeCalled`.
But I leave it upto you to accept/ignore my comment.